lib/components/narrative/metro/default-route-renderer.tsx

```diff
@@ -2,16 +2,21 @@ import { Leg } from '@opentripplanner/types'
 import React from 'react'
 import styled from 'styled-components'
 
-// TODO: This should not be a section, but using div/span
-// doesn't allow us to style it from within the RouteBlock
-const Block = styled.section<{ color: string }>`
+const Block = styled.span<{ color: string }>`
   background: #${(props) => props.color}1A;
-  padding: 3px 7px;
-  border-top: 5px solid #${(props) => props.color};
   border-radius: 5px;
-  text-align: center;
+  border-top: 5px solid #${(props) => props.color};
   display: inline-block;
-  max-width: 12vw;
+  margin-top: -2px;
+  padding: 3px 7px;
+
+  /* Below is for route names that are too long: cut-off and show ellipsis. */
+  max-width: 150px;
+  overflow: hidden;
```

Comment:
Not sure if it's this line, but there is some trouble in the itinerary body that wasn't there previously.
<img width="204" alt="Screenshot 2022-12-06 at 9 58 50 AM" src="https://user-images.githubusercontent.com/86619099/205946849-ae2518b5-9c2d-4eb3-91b5-8c2391edeef7.png">

Reply:
Is this on Safari? This also appears in the dev branch.

Reply:
Might've been a cache issue when I switched back to dev to check. If it's not a regression then we can leave as is

Reply:
That probably comes from this line: https://github.com/opentripplanner/otp-react-redux/blob/20ca3fa4531a28a9a7d0c0fb2101f0db9a1c03a4/lib/components/narrative/metro/route-block.tsx#L30